docs/getting-started/browsing-gno-source-code.md

```diff
@@ -0,0 +1,96 @@
+---
+id: browsing-gno-source-code
+---
+
+# Browsing Gno Source Code
+
+## Overview
+
+In this tutorial, you will learn how to browse deployed Gno [Realms](../explanation/realms.md)
+and [Packages](../explanation/packages.md). Additionally, you will understand how the `Render` method is utilized
+to achieve Realm state visibility.
+
+## Prerequisites
+
+- **`gnoweb` set up. Reference the [Installation](installation.md#3-installing-other-gno-tools) guide for steps**
+
+## 1. Start the local chain
+
+In order for `gnoweb` to fetch realm and package source code, it needs to connect a running Gno node. For a better
+overview on running a local node, please reference the [Starting a Local Chain](setting-up-a-local-chain.md) guide.
+
+In this guide, we will start a local node with the default configuration. Navigate to the `gno.land` sub-folder and run:
+
+```bash
+gnoland start
+```
+
+## 2. Start `gnoweb`
+
+Now that the chain is running, we can start the `gnoweb` tool:
+
+```bash
+gnoweb
+```
+
+:::info Different JSON-RPC URL
```

Comment:
Is this format a standard in Docusaurus?

It would be great to share it with @alexiscolin and #902.

Reply:
Yes, it's part of Docusaurus for sprucing up Markdown:

https://docusaurus.io/docs/markdown-features/admonitions